draft-ietf-ppm-dap.md

```diff
@@ -621,6 +621,8 @@ number generator. Each task has the following parameters associated with it:
   task must have.
 * `max_batch_lifetime`: The maximum number of times a batch of reports may be
   queried by the Collector.
+* `max_task_lifetime`: The maximum timestamp of a task, after which the aggregators
```

Comment:
"maximum timestamp of a task" is a bit ill-defined. Can you be more precise about what this means? Is it that reports with timestamps exceeding the max_task_lifetime are rejected? Or is it that requests are aborted if they arrive after max_task_lifetime?